src/support/serde.rs

```diff
@@ -106,20 +106,28 @@ impl<'de, const BITS: usize, const LIMBS: usize> Deserialize<'de> for Bits<BITS,
     }
 }
 
-/// Serde Visitor for human readable formats
-struct StrVisitor<const BITS: usize, const LIMBS: usize>;
+/// Serde Visitor for human readable formats.
+///
+/// Accepts either a primitive number, a decimal or a hexadecimal string.
+struct HRVisitor<const BITS: usize, const LIMBS: usize>;
```

Comment:
nit: rust naming convention is to only capitalize first letter in acronyms. `HrVisitor`